src/domUpdates.js

```diff
@@ -219,10 +231,13 @@ const closeRecipe = () => {
   clickedRecipe.classList.add("hidden");
   clickedRecipe.classList.remove("flex");
   clickedRecipe.classList.remove("fade-in");
-  ingredientsList.innerHTML = '';
 };
 
-const getIngredients = currentRecipeCard => {
+const populateIngredients = currentRecipeCard => {
+  console.log(currentRecipeCard)
```

Comment:
Not a blocker but did you miss removing the console log? 
If nothing else, could you comment it out for now?

Reply:
I sure did miss that!  Done.